randobot/handler.py

```diff
@@ -165,6 +170,14 @@ async def chat_message(self, data):
 
     async def race_data(self, data):
         await super().race_data(data)
+        if self._race_pending() and self.state.get('password_active') and not self.state['password_published']:
+            seed_password = self.zsr.get_password(self.state['seed_id'])
+            await self.set_bot_raceinfo('File Select Password: %{seed_password} -- Hash: %(seed_hash)s\n%(seed_url)s' % {
+                'seed_password': seed_password,
+                'seed_hash': self.state['seed_hash'],
+                'seed_url': self.seed_url % self.state['seed_id'],
+            })
+            self.state['password_published'] = True
```

Comment:
I'd recommend sending the password as a chat message as well as putting it in the info, for anyone who's using the popout chat window (or is on mobile)